Aquifer/NSFileHandleExt.m

```diff
@@ -15,129 +15,140 @@
 #define BUFFER_SIZE 256
 
 in_addr_t getipaddr(const char *host) {
-    /* let's look it up */
```

Comment:
Good god, what is this stuff?  I'm going to see what I can do about this weirdness.